cfgov/jinja2/v1/mmt-cost-of-credit-tool/alert-icon.html

```diff
@@ -0,0 +1,19 @@
+{% import 'success-icon.html' as success_icon %}
```

Comment:
How many of these Jinja templates does your app actively use? I remember we copied a bunch of these over from an existing app, and we likely don't need most of them. This feedback applies to every file below this one ending in ".html" too

Reply:
I think I cleared everything out.